src/pages/index.tsx

```diff
@@ -11,12 +14,15 @@ export const getStaticProps: GetStaticProps = async ({ locale }) => ({
 
 const Home = function (_props: InferGetStaticPropsType<typeof getStaticProps>) {
   const { t } = useTranslation('common');
+  const [blogs] = useState(blogsData);
```

Comment:
you are not using the useState in a good way because in here it is useless you can directly use the data that you imported from the blogData